src/api/ledger/parse/utils.js

```diff
@@ -62,8 +62,8 @@ function parseOutcome(tx: Object): ?Object {
     fee: utils.common.dropsToXrp(tx.Fee),
     balanceChanges: balanceChanges,
     orderbookChanges: orderbookChanges,
-    ledgerVersion: tx.ledger_index,
-    indexInLedger: tx.meta.TransactionIndex
+    ledgerVersion: tx.ledger_index || 0,
```

Comment:
Originally I've added this because I've removed checks from `compareTransactions` function.
If we sure what this field always will be present we can remove this.

BTW, can unvalidated transactions be part of `getTransactions` response? In that case `compareTransactions` will throw exception.